src/actions/actionTypes.js

```diff
@@ -75,6 +75,7 @@ export const CLUSTER_DELETE_REQUEST = 'CLUSTER_DELETE_REQUEST';
 export const CLUSTER_DELETE_CONFIRMED = 'CLUSTER_DELETE_CONFIRMED';
 export const CLUSTER_DELETE_SUCCESS = 'CLUSTER_DELETE_SUCCESS';
 export const CLUSTER_DELETE_ERROR = 'CLUSTER_DELETE_ERROR';
+export const CLUSTER_REMOVE_FROM_STORE = 'CLUSTER_REMOVE_FROM_STORE';
```

Comment:
fyi @ograu I introduced `CLUSTER_REMOVE_FROM_STORE ` because `CLUSTER_DELETE_REQUEST` is actually the trigger to show the cluster delete modal.

We might have to rename `CLUSTER_DELETE_REQUEST` to align it with what the `_REQUEST` actions actually mean now, but for now this was my short path to fixing this bug.

Reply:
👍